src/python/singa/__init__.py

```diff
@@ -1,3 +1,242 @@
-def main():
-    """Entry point for the application script"""
-    print("Welcome to SINGA!")
+#!/usr/bin/env python
+# encoding: utf-8
+#/************************************************************
+#*
+#* Licensed to the Apache Software Foundation (ASF) under one
+#* or more contributor license agreements.  See the NOTICE file
+#* distributed with this work for additional information
+#* regarding copyright ownership.  The ASF licenses this file
+#* to you under the Apache License, Version 2.0 (the
+#* "License"); you may not use this file except in compliance
+#* with the License.  You may obtain a copy of the License at
+#*
+#*   http://www.apache.org/licenses/LICENSE-2.0
+#*
+#* Unless required by applicable law or agreed to in writing,
+#* software distributed under the License is distributed on an
+#* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#* KIND, either express or implied.  See the License for the
+#* specific language governing permissions and limitations
+#* under the License.
+#*
+#*************************************************************/
+
+#*sudo apt-get install libjpeg-dev
```

Comment:
pls to use block comments with details
the author names are recorded by github history.
copyright is specified in https://github.com/apache/incubator-singa/blob/master/NOTICE
contact should be dev email.